### PR TITLE
Rename module alluxio-ui to webui for consistency

### PR DESCRIPTION
**Pull request by apc999**
This pull request simply renames top level module `alluxio-ui` to `webui` to keep it consistent with other modules inside Alluxio

### CODE REVIEW

**Comment by calvinjia** [comment]
@apc999 Thanks for renaming this. You will need to change the `WEB_RESOURCES` property key and the `contributor-tools` and other docs.

**Comment by apc999** [comment]
PTAL @calvinjia

File: webui/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.alluxio</groupId>
-  <artifactId>alluxio-ui</artifactId>
+  <artifactId>alluxio-webui</artifactId>
```
Comment (calvinjia) [inline webui/pom.xml]: Should we just call this `webui`?

Comment (apc999) [reply webui/pom.xml]: our POM artifact convention is to make `foo/` into `alluxio-foo`, like `integration` module:
` <artifactId>alluxio-integration</artifactId>`

Comment (calvinjia) [reply webui/pom.xml]: got it, thanks.

**Comment by calvinjia** [comment]
alluxio-bot, merge this please.

**Pull request merged**